sensor-validate.c

```diff
@@ -1,28 +1,69 @@
 #include "sensor-validate.h"
 
-int _give_me_a_good_name(double value, double nextValue, double maxDelta) {
+
+/*---------------------------------------------------------------------------*/
+/*     FUNCTION:    IsThereSuddenJump
+ */
+/*!    \brief      to check if there is a sudden junmp
+ * 
+ *     \param       current value ,next value and max value
+ *     \returns     the status if the jump was abrupt
+ *
+*//*------------------------------------------------------------------------*/
+int IsThereSuddenJump(double value, double nextValue, double maxDelta) {
```

Comment:
A reader would expect this function to return 1 when there is a sudden jump. But it is the other way round.